http_service/Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN pip install --disable-pip-version-check --quiet --no-cache-dir -r /requireme
 COPY . /code/http_service
 RUN pip install --disable-pip-version-check --quiet --no-cache-dir /code/http_service
 
-CMD gunicorn -b 0.0.0.0:$PORT bugbug_http.app --preload --timeout 30 -w 3
+# Run the Pulse listener in the background
+CMD (/code/http_service/listener.py &> /code/http_service/pulse.log &) && gunicorn -b 0.0.0.0:$PORT bugbug_http.app --preload --timeout 30 -w 3
```

Comment:
We should find a way to log to the foreground rather than a file

Reply:
The whole problem I was facing had to do that in order for new code to be executed I needed to call `docker-compose build bugbug-http-service`.

Redirection to disk was left by mistake. Fixed locally.